### DISCUSSION TITLE
Add Grouping?

### DISCUSSION THREAD

**Discussion by TheFoggyRoad** [dfinke/ImportExcel#1381] [Q&A] [answered]
Hi All. Thank you for such a wonderful PS module. 

I was wondering if there is a way to add grouping (Outline) to the exported spreadsheet?

**Comment by dfinke** [comment, marked as answer]
Not sure, are you looking for something like this? https://github.com/dfinke/ImportExcel/issues/556

**Comment by TheFoggyRoad** [comment]
Thank you! I searched Discussions but forgot to search Issues

**Reply by dfinke** [reply]
Good, glad that worked
